server/src/test/java/org/candlepin/dto/manifest/v1/BrandingTranslatorTest.java

```diff
@@ -66,8 +60,6 @@ protected void verifyOutput(Branding source, BrandingDTO dest, boolean childrenG
             assertEquals(source.getProductId(), dest.getProductId());
             assertEquals(source.getName(), dest.getName());
             assertEquals(source.getType(), dest.getType());
-            assertEquals(source.getUpdated(), dest.getUpdated());
-            assertEquals(source.getCreated(), dest.getCreated());
```

Comment:
The created/updated field testing should not be removed. They were accidentally not tested in the temporary ProductBrandingTranslatorTest.java file. In fact, they should also be added in the api/v1/BrandingTranslatorTest.java now that this was noticed.